src/theme/includes/scripts.html

```diff
@@ -1,9 +1,16 @@
 <script>
-  window.addEventListener('{{ siteName | trim }}_pageview', function () {
+  var onClientSideRouteChange = function () {
     /* Fire off an event to your own tracking
        service to log a pageview from here
+       console.log('PAGE VIEW HAPPENED'); */
 
-       console.log('PAGE VIEW HAPPENED');
-    */
-  });
+    /* Dispatch ad reload when route
+       changes client-side */
+    loadAds();
+  };
+
+  var loadAds = function () {};
```

Comment:
Do you think we should have empty implementation examples in the code base of Iris? Maybe consider moving this to the Petitio fork?